shell/android-studio/reicast/src/main/java/com/reicast/emulator/FileBrowser.java

```diff
@@ -126,61 +57,6 @@ public void onCreate(Bundle savedInstanceState) {
 		return out;
 	}
 
-	// Container Activity must implement this interface
```

Comment:
It's already scanning the Content Locations (and subfolders) for games.

What's missing is:
- Asynchronous threaded scan to avoid freezing the UI on slow media/machines
- A Refresh/Rescan button or gesture
- Game metadata/covers download